test/integration/targets/k8s/runme.sh

```diff
@@ -21,5 +21,5 @@ ansible-playbook -v playbooks/merge_type_fail.yml "$@"
 # Run full test suite
 virtualenv --system-site-packages --python "${PYTHON}" "${MYTMPDIR}/openshift-recent"
 source "${MYTMPDIR}/openshift-recent/bin/activate"
-$PYTHON -m pip install 'openshift==0.7.2'
+$PYTHON -m pip install 'openshift>=0.8.1'
```

Comment:
It's OK to update this to a newer version, but it should remain pinned to a specific version.

Having Ansible use the latest version available results in spontaneous test failures when external resources change in incompatible ways. This isn't specific to openshift, but is a general problem with integration tests.

It is important that our tests remain stable so that we do not negatively impact the regular development and release processes. That being said, it would be a good idea to periodically open a PR to update the version to the latest available release and resolve any issues that doing so may expose.

In the future we may have a mechanism for running tests on a scheduled basis that automatically use newer versions of dependencies to detect issues early, but that isn't something we should be doing during our regular test runs.

Reply:
@fabianvf Would you be able to take a look at this, thanks in advance

Reply:
@gundalow By this, do you mean this version pin, or updating tests to periodically run with unpinned versions? The first is already resolved and I don't quite have the bandwidth for the second currently.